protostar/starknet/cheaters/utils.py

```diff
@@ -0,0 +1,24 @@
+import os
+from typing import Optional
+
+from protostar.protostar_exception import ProtostarException
+from protostar.starknet.address import Address
+from protostar.starknet.data_transformer import CairoOrPythonData
+
+
+def assert_no_expected_calls(
```

Comment:
Not sure if this place is right for this function. I tried a couple of different places but I was getting a lot of complaints about circular dependencies.

Reply:
oh we don't like utils modules around here :P let's name this file something else.

Reply:
Honestly, the name is not a problem. It can be anything, really. If the fact that this function is in a separate file and it lies in this directory is okay then we're good to go I think.

Reply:
I don't think it should be located in this module, this is not a general util function but bussines logic which is reused in two places. I would create `ExpectCheaters`, put logic there and include this function there, WDYT?

Reply:
[add 'expects' cheater, modify cheaters hierarchy](https://github.com/software-mansion/protostar/pull/1363/commits/650efd7813074d4654e979c9a21a14da3dffad95)